packages/twenty-server/.env.example

```diff
@@ -73,3 +73,7 @@ SIGN_IN_PREFILLED=true
 # MUTATION_MAXIMUM_RECORD_AFFECTED=100
 # CHROME_EXTENSION_ID=bggmipldbceihilonnbpgoeclgbkblkp
 # PG_SSL_ALLOW_SELF_SIGNED=true
+# OPENAI_API_KEY=sk-proj-abcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcdabcd
+# LANGFUSE_SECRET_KEY=sk-lf-abcdabcd-abcd-abcd-abcd-abcdabcdabcd
+# LANGFUSE_PUBLIC_KEY=pk-lf-abcdabcd-abcd-abcd-abcd-abcdabcdabcd
```

Comment:
Related to comment above, we should be able to use the application without langfuse ; I know you get the prompt from there so that might be an issue. Usually we have a driver-based approach. We don't want the self-hosting experience to rely on external paid tools